### PR TITLE
AR Number Range - Spec Update

### PR DESCRIPTION
**Pull request by neuarjunrpanicker**
PR Includes

Specs
Number Range Model - Proposed Enhancements

• Proposed Enhancement 22 : Extractor Passed - 8, Full Passed - 1, Skipped - 14
• Total Test Cases 167: Extractor Passed - 113, Full Passed - 49, Skipped - 54
• New Cases 100: Extractor Passed - 72, Full Passed - 13, Skipped- 28
• Existing Cases 67: Extractor Passed - 41, Full Passed - 36, Skipped - 26

Checklist
- [x] Verified inputs from Linguist
- [x]  Verified for consistency of similar cases given by linguist
- [x]  Categorized the test spec
- [x]  No duplicate inputs
- [x]  Verified indexes
- [x]  Executed the ./build.cmd in VScode for duplication
- [x]  Updated daily status tracker in teams
- [x]  Uploaded the JSON and excel sheets in Teams folder under Finalized Specs

Removed unwanted spaces and corrected indexes for the cases.

### CODE REVIEW

File: Specs/Number/Arabic/NumberRangeModel.json [hunk 1]
```diff
@@ -1508,66 +1508,66 @@
...
   },
   {
-    "Input": "ماذا عن أكثر من ٢  في ٥ ",
+    "Input": "ماذا عن أكثر من ٢ في ٥",
     "NotSupportedByDesign": "java, javascript, python",
     "NotSupported": "dotnet",
     "Results": [
       {
-        "Text": "أكثر من ٢ ",
+        "Text": "أكثر من ٢",
```
Comment (neudurgeshp) [inline Specs/Number/Arabic/NumberRangeModel.json:1570]: Why start end is not there?

File: Specs/Number/Arabic/NumberRangeModel.json [hunk 2]
```diff
@@ -1578,12 +1578,12 @@
     ]
   },
   {
-    "Input": "هل يمكنك أن تعرضني سجلات أكثر من ٣٠٠٠٠  في عام ٢٠٠٩م",
+    "Input": "هل يمكنك أن تعرضني سجلات أكثر من ٣٠٠٠٠ في عام ٢٠٠٩م",
```
Comment (neudurgeshp) [inline Specs/Number/Arabic/NumberRangeModel.json:1581]: Same and so on

File: Specs/Number/Arabic/NumberRangeModel.json [hunk 3]
```diff
@@ -1594,12 +1594,12 @@
     ]
   },
   {
-    "Input": "هل يمكنك أن تعرضني سجلات أقل من ٣٠٠٠٠  في عام ٢٠٠٩م ",
+    "Input": "هل يمكنك أن تعرضني سجلات أقل من ٣٠٠٠٠ في عام ٢٠٠٩م",
```
Comment (neudurgeshp) [inline Specs/Number/Arabic/NumberRangeModel.json:1597]: Same and so on

Comment (neudurgeshp) [reply Specs/Number/Arabic/NumberRangeModel.json:1597]: @neuarjunrpanicker why we missed start end in most of the cases. Fix that update PR again

Comment (neuarjunrpanicker) [reply Specs/Number/Arabic/NumberRangeModel.json:1597]: @neudurgeshp for these cases, there is no change in start end from previous.

**Comment by neudurgeshp** [comment]
@neuarjunrpanicker as discussed Resolve conflict and Raise a fresh PR

**Pull request closed without merging**